.github/workflows/tests.yml

```diff
@@ -2,9 +2,6 @@ name: Tests
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - master
```

Comment:
I agree with the logic here: if we just ran the test suite on an up-to-date PR before merging into master, there's almost certainly no need to re-run it on master. The very unlikely exceptions will get caught by the daily run anyway.

However, should we update all the `if: ${{ ... || github.event_name == 'push') }}` clauses to remove the references to push events, since they no longer exist?

Reply:
Ah, of course! Good catch.